app/models/split.rb

```diff
@@ -82,8 +119,34 @@ def create_decision!(params = {})
     build_decision(params).tap(&:save!)
   end
 
+  def registry
+    if try(:feature_incomplete?)
```

Comment:
Is this shorthand for `respond_to?(:feature_complete?) && feature_incomplete?`. Might be worth adding a comment explaining that this field is injected by one of the scopes.

Reply:
Yeah. And yeah!